dockers/dev/README.md

```diff
@@ -4,13 +4,13 @@
 
 `vald-dev-container` is designed for the development of Vald on Docker.
 
-This image includes some libraries required for implementation and is based on `ubuntu:devel` image.
+This image includes some libraries required to develop Vald and is based on `mcr.microsoft.com/vscode/devcontainers/go:1` image.
```

Comment:
It would be better to show the specific URL of `mcr.microsoft.com/vscode/devcontainers/go:1`